theme/static/css/hydroshare_core.css

```diff
@@ -682,19 +712,44 @@ li.account p{
 	text-transform: none;
 	word-wrap: break-word;
 }
+
 #cogs-menu,
 #profile-menu {
     display: none;
     text-transform: initial;
 }
+
 #profile-menu > .dropdown-toggle {
-    font-size: 14px;
+    padding: 0;
+}
+
+#profile-menu .dropdown-footer {
+    position: absolute;
+    bottom: 20px;
+    border-top: 1px solid #CCC;
+    width: 100%;
+    margin: -20px;
+	padding: 10px;
+	background: #DDD;
+	-webkit-border-bottom-right-radius: 4px;
+	-webkit-border-bottom-left-radius: 4px;
+	-moz-border-radius-bottomright: 4px;
+	-moz-border-radius-bottomleft: 4px;
+	border-bottom-right-radius: 4px;
+	border-bottom-left-radius: 4px;
```

Comment:
Is there a reason some properties are tabbed over extra? I'm not one to tell you to fix it (since I'm not really a css expert), unless you think you should and just missed it. I'm really just wondering.